packages/dataclass-editor/src/components/dataclass/detail/FieldDetailContent.tsx

```diff
@@ -26,28 +39,36 @@ export const FieldDetailContent = () => {
       <BasicField label='Type'>
         <Input value={field.type} onChange={event => handleFieldPropertyChange('type', event.target.value)} />
       </BasicField>
-      <BasicCheckbox
-        label='Persistent'
-        checked={field.modifiers.includes('PERSISTENT')}
-        onCheckedChange={event => {
-          let newModifiers = structuredClone(dataClass.fields[selectedField].modifiers);
-          if (event.valueOf()) {
-            newModifiers.push('PERSISTENT');
-          } else {
-            newModifiers = newModifiers.filter(modifier => modifier !== 'PERSISTENT');
-          }
-          handleFieldPropertyChange('modifiers', newModifiers);
-        }}
-      />
       <BasicField label='Comment'>
         <Textarea value={field.comment} onChange={event => handleFieldPropertyChange('comment', event.target.value)} />
       </BasicField>
-      <BasicField label='Annotations'>
-        <Textarea
-          value={field.annotations.join('\n')}
-          onChange={event => handleFieldPropertyChange('annotations', removeEmptyStrings(event.target.value.split('\n')))}
-        />
-      </BasicField>
+      <Collapsible>
+        <CollapsibleTrigger>Properties</CollapsibleTrigger>
+        <CollapsibleContent>
+          <BasicCheckbox
+            label='Persistent'
+            checked={field.modifiers.includes('PERSISTENT')}
+            onCheckedChange={event => {
+              let newModifiers = structuredClone(field.modifiers);
+              if (event.valueOf()) {
+                newModifiers.push('PERSISTENT');
+              } else {
+                newModifiers = newModifiers.filter(modifier => modifier !== 'PERSISTENT');
+              }
+              handleFieldPropertyChange('modifiers', newModifiers);
+            }}
+          />
+        </CollapsibleContent>
+      </Collapsible>
```

Comment:
The properties (modifiers) will always be closed by default. Once we have a good default behavior, we can check it to decide whether it is open.